scripts/run-para-test-single.sh

```diff
@@ -0,0 +1,22 @@
+#/bin/sh
+
+# If you are in the tests folder, go back to the root of the repository
+FOLDER="${PWD##*/}"
+if [[ "$FOLDER" == "tests" ]]; then
+  cd ..
+fi
+
+# Get relay binary
+source scripts/get-alphanet-relay-image.sh
```

Comment:
why do you source it instead of simply executing it ?

Reply:
By habit I always use source to make sure I'm in the same process context

Reply:
In the case of source, it will import the environment variable set by the script, which might not be good (for readability)

Reply:
fixed